squirrel_interaction/board/src/teleop.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+
+""" The JoyController Node serves as an example client for the interaction components of SQUIRREL
+
+    By subscribing to the /Joy [Joy] topic, this node allows the user to control:
+        all motors of the interaction board.
+        base leds,
+        mouth leds,
+        screen,
+        speakers.
+
+"""
+
+import rospy
+from sensor_msgs.msg import Joy, JointState
+from std_msgs.msg import Float64
+from geometry_msgs.msg import Twist
+from math import degrees, radians
+from raspberry_screen.srv import *
```

Comment:
Yes, you can help me restructure the packages. 

I'll get right on the other raspberry_screen problem, must have been a commit issue.